src/components/SignatureRequest.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+import { useWallet } from '../providers/Wallet'
+import { getProviderFromUseWalletId } from '../ethereum-providers'
+import { signMessage } from '../lib/web3-utils'
+import { dayjs } from '../utils/date-utils'
+
+import signRequestSuccessIllustration from '../../src/assets/signRequestSuccess.svg'
+import signRequestFailIllustration from '../../src/assets/signRequestFail.svg'
+import signProcessing from '../../src/assets/loader.gif'
+
+const SignerRequest = React.memo(function SignerRequest({
+  actionText,
+  compactMode,
+  onActionSuccess,
+  onSignSuccess,
+  successText,
+  title,
+}) {
+  const [signingError, setSigningError] = useState(false)
+  const [signHash, setSignHash] = useState('')
```

Comment:
Perhaps we can use a boolean for this state instead, and use a name like `signedHash`, since we never need the actual signature in this component.

Reply:
yes we are using it https://github.com/aragon/court-dashboard/blob/90a4d954960d1a47bee4ea0dc656ac70b470721d/src/components/SignatureRequest.js#L81

Reply:
But there we're not using the state variable... we're using the local variable in the function closure.

Reply:
ahhh hah true